modules/@apostrophecms/asset/index.js

```diff
@@ -554,6 +554,16 @@ module.exports = {
       // If you are trying to enable IE11 support for ui/src, use the
       // `es5: true` option (es5 builds are disabled by default).
       configureBuilds() {
+        const srcPrologue = stripIndent`
```

Comment:
Moved this prologue from public to src because src loads first (as it must, if custom webpack builds are going to use apos frontend utilities) and src now contains our core frontend js